Public/Src/App/Bxl/ServerDeployment.cs

```diff
@@ -174,12 +145,22 @@ public static string ComputeDeploymentDir(string serverDeploymentRoot)
         internal static void KillServer(string serverDeploymentRoot)
         {
             // Check if the main root process (likely bxl.exe) is in use before attempting to delete, so we avoid partially deleting files
-            // Not completey bullet proof (there can be a race) but it is highly unlikely the process starts to be in use right after this check
+            // Not completely bullet proof (there can be a race) but it is highly unlikely the process starts to be in use right after this check
             Assembly rootAssembly = Assembly.GetEntryAssembly();
             Contract.Assert(rootAssembly != null, "Could not look up entry assembly");
 
             string assemblyFullPath = Path.Combine(serverDeploymentRoot, new FileInfo(AssemblyHelper.GetAssemblyLocation(rootAssembly)).Name);
 
+#if FEATURE_CORECLR

```

Comment:
Delete lines 149-152, replace
```csharp
string assemblyFullPath = Path.Combine(serverDeploymentRoot, new FileInfo(AssemblyHelper.GetAssemblyLocation(rootAssembly)).Name);
```
with
```csharp
string assemblyFullPath = Path.Combine(serverDeploymentRoot, new FileInfo(AssemblyHelper.GetThisProgramExeLocation()).Name);
```
then delete this whole block,

Reply:
yay - forgot about that one thanks!